website/blog/2024-10-23-release-0.76-new-architecture.md

```diff
@@ -0,0 +1,224 @@
+
+### Updates to Minimum iOS and Android SDK requirements
+
+We have updated our minimum platform and SDK versions:
+
+- iOS: [15.1](https://support.apple.com/en-gb/108051#151)
+- Android: [SDK 24](https://developer.android.com/tools/releases/platforms#7.0) (Android 7)
+
+This change was announced earlier in the year when 0.75 [was released](/blog/2024/08/12/release-0.75#last-version-supporting-minsdk-23-and-miniosversion-134). For more background, please see the dedicated posts for [Android](https://github.com/react-native-community/discussions-and-proposals/discussions/802) and [iOS](https://github.com/react-native-community/discussions-and-proposals/discussions/812).
+
+### Other Breaking Changes
+
+- **Animation**
+  - Stop sending state updates to React in looping animation ([commit](https://github.com/facebook/react-native/commit/8af5e893c89b24dca972afe36ef442fd509d5804)). This was causing unnecessary re-rendering with looping animations.
+- **devtools:**
+  - Remove Inspector Panel perf + network tabs under New Arch ([commit](https://github.com/facebook/react-native/commit/f220bde4d7d556b3be83cccff0eec5da22921aba)). ([RFC](https://github.com/react-native-community/discussions-and-proposals/blob/main/proposals/0777-remove-legacy-element-inspector-features.md))
+- **text engine**
+  - Always use AttributedStringBox instead of AttributedString in TextLayoutManager ([commit](https://github.com/facebook/react-native/commit/ee597bfe2bf7f11ac60dab87f47dbcb6e840749c))
+
+#### Android
+
+- **rendering:**
+  - View backgrounds are no longer directly `ReactViewBackgroundDrawable` or `CSSBackgroundDrawable` ([commit](https://github.com/facebook/react-native/commit/5f14963da081deeac2fb540d58c8667c2d2c5d08))
+
+#### iOS
+
+- **turbomodule**
+  - Removed the `RCT_EXPORT_CXX_MODULE_EXPERIMENTAL` Macro for autolinking pure Cxx module ([commit](https://github.com/facebook/react-native/commit/184646e49169110842bafa6c02e52f6eec1e0ffa#diff-28b37f7a1a36054d37cd957bd4718ce481006efb286e5e88dec565087c4a8727)).
+
+## Acknowledgements
+
+React Native 0.76 contains over [1052 commits](https://github.com/facebook/react-native/compare/6f7eae5c583090bd62ca60a4c3170445e5a0a751..0.76-stable) from 156 contributors. Thanks for all of your hard work!
+
+Thanks to all the additional authors that worked on documenting features in this release post:
+
+- [Joe Vilches](https://github.com/joevilches) and [Nick Gerleman](https://github.com/NickGerleman) box-shadow and filter style props.
+- [Alex Hunt](https://github.com/huntie) React Native DevTools.
+- [Nicola Corti](https://github.com/cortinico) Android shipping as a single library.
+
+## Upgrade to 0.76
+
+Please use the [React Native Upgrade Helper](https://react-native-community.github.io/upgrade-helper/) to view code changes between React Native versions for existing projects, in addition to the Upgrading docs.
+
+To create a new project:
```

Comment:
probably need to be clear that this isn't the official way to create a project, it's the community cli. 

And since we recommend people create new projects with Expo, I'd pull up the Expo line to be first

Reply:
The problem is that Expo is not ready yet to support 0.76. We need to wait for SDK 52. We can move up the expo sentence.